apps/filecoin-site/src/app/store-data/constants/serviceStructuredData.ts

```diff
@@ -0,0 +1,19 @@
+import { PATHS } from '@/constants/paths'
+import { BASE_URL } from '@/constants/siteMetadata'
+import { WEBSITE_SCHEMA } from '@/constants/structuredDataConstants'
+
+export const SERVICE_STRUCTURED_DATA = {
+  serviceType: 'Decentralized data storage',
+  providerId: WEBSITE_SCHEMA.publisher['@id'],
+  areaServed: 'Global',
+  termsOfService: `${BASE_URL}/terms`,
```

Comment:
Is this supposed to be:

```suggestion
  termsOfService: `${BASE_URL}{PATHS.TERMS_OF_USE.path`,
```